gst/nnstreamer/tensor_if/gsttensorif.c

```diff
@@ -1097,6 +1098,43 @@ gst_tensor_if_calculate_cv (GstTensorIf * tensor_if, GstBuffer * buf,
   return TRUE;
 }
 
+/**
+ * @brief Registers a callback for tensor_if custom condition
+ * @return TRUE if there is no error.
+ */
+gboolean
+tensor_if_custom_register (GstElement * tensorif_h, tensor_if_custom func,
+    void *data)
+{
+  GstTensorIf *tensor_if = (GstTensorIf *) tensorif_h;
+
+  tensor_if->custom = g_new0 (custom_cb_s, 1);
+
+  if (NULL == tensor_if->custom) {
```

Comment:
This is unnecessary, and the following lines are never reached.

Reply:
Right. Updated